app/controllers/dealerships_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class DealershipsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
this method is being defined basically in every controller. You can safely use it on `ApplicationController` so it will work the way you want.

Reply:
fixed on #4